docs/faq.md

```diff
@@ -116,4 +116,67 @@ cite_assay@meta.features <- cite_row_metadata
 
 # add altExp from SingleCellExperiment as second assay to Seurat
 seurat_object[["CITE"]] <- cite_assay
-```
\ No newline at end of file
+```
+
+## What if I want to use Python instead of R? 
+
+We provide single-cell and single-nuclei gene expression data as RDS files, which must be opened in R to view the contents. 
+If you prefer to work in Python, there are a variety of ways of converting the count data to Python-compatible formats.
+We have found that one of the more efficient is conversion via the 10X format using [`DropletUtils::write10xCounts()`](https://rdrr.io/bioc/DropletUtils/man/write10xCounts.html). 
+Note that you will need to install the [`DropletUtils` package](https://www.bioconductor.org/packages/devel/bioc/html/DropletUtils.html) to use this function.
+
+This will output three files to a new directory:
```

Comment:
Maybe we want to be a bit more specific here, as there are a number output formats we could apply. 

This maybe gets the info we need, without the details about .gz or not .gz?
```suggestion
When used as described below, `DropletUtils::write10xCounts()` will output three files to a new directory, following the format used by Cell Ranger 3.0 (and later):
```